esrally/telemetry.py

```diff
@@ -324,6 +324,7 @@ def __init__(self, log_root):
 
     def detach_from_node(self, node, running):
         if running:
+            io.ensure_dir(self.log_root)
```

Comment:
see `JitCompiler.instrument_java_opts` and `Gc.instrument_java_opts` for analogous usage